keda-2.13.yaml

```
@@ -0,0 +1,96 @@
+# KEDA 2.11 version stream
+# Remove after 2.14 or 3.0
+package:
+  name: keda-2.13
+  version: 2.13.1
+  epoch: 0
+  description: KEDA is a Kubernetes-based Event Driven Autoscaling component. It provides event driven scale for any container running in Kubernetes
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - tzdata
+    provides:
+      - keda=${{package.full-version}}
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - go
+      - protobuf-dev
+      - protoc
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 41dd7a8558fdb274f2c0861f6c6d1eba01f2dcd5
+      repository: https://github.com/kedacore/keda
+      tag: v${{package.version}}
+
+  # CVE-2023-39325 CVE-2023-45142 CVE-2023-47108
+  - uses: go/bump
+    with:
+      deps: sigs.k8s.io/controller-tools@v0.14.0 golang.org/x/crypto@v0.19.0 github.com/go-jose/go-jose/v3@v3.0.3 github.com/cloudflare/circl@v1.3.7 go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp@v0.46.0 go.opentelemetry.io/otel/exporters/otlp/otlptrace@v1.19.0 go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc@v1.19.0 go.opentelemetry.io/otel/sdk@v1.21.0 go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc@v0.46.0
+
+  - uses: patch
+    with:
+      patches: kube-openapi.patch
+
+  - runs: |
+      go clean -cache -modcache
```

```suggestion
      deps: github.com/cloudevents/sdk-go/v2@v2.15.2 github.com/go-jose/go-jose/v3@v3.0.3

  - runs: |
```

For the new version we do not need any of this